src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/PosteriorLikelihoodsUtilsUnitTest.java

```diff
@@ -0,0 +1,485 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+@SuppressWarnings("unchecked")
+public final class PosteriorLikelihoodsUtilsUnitTest extends BaseTest {
+
+    Allele Aref, T, C, G, Cref, ATC, ATCATC;
+
+    @BeforeSuite
+    public void setup() {
+        // alleles
+        Aref = Allele.create("A", true);
+        Cref = Allele.create("C", true);
+        T = Allele.create("T");
+        C = Allele.create("C");
+        G = Allele.create("G");
+        ATC = Allele.create("ATC");
+        ATCATC = Allele.create("ATCATC");
+    }
+
+    private String arraysEq(final int[] a, final int[] b) {
```

Comment:
hmn, I thought this was outdated, but it looks like there still isn't an implementation of this in testng